test/e2e/specs/onboarding/vertical__none.ts

```diff
@@ -0,0 +1,140 @@
+/**
+ * @group calyepso-quarantined
```

Comment:
**issue**: This isn't a real group - for quarantined tests, we use `quarantined`.

Consider changing it to either `calypso-pr` to run for every commit on every branch or `calypso-release` to be run only for the smoke/canary tests when a PR is merged.

**question**: Given the information above - what group would you choose, and why?

Reply:
To be honest, I'm not sure how that group got there (complete with an embarrassing typo). I changed all of the groups to `calypso-pr` based on your description above, mostly because I don't see my tests as being smoke/canary anymore. `vertical__selected.ts` could have been `calypso-release` at the beginning when it was a simple end-to-end flow, but now there's a bit more test case checking in there. If you feel differently, please let me know!